### DISCUSSION TITLE
Question:How polygon edge uses checkpointing?

### DISCUSSION THREAD

**Discussion by Kvenessa** [0xPolygon/polygon-edge#451] [Q&A] [answered]
How polygon edge allows for checkpointing of a chain created with polygon edge?  

chain build with polygon edge local setup, what additional setup required for chain which will checkpoint on parent chain (say polygon) to leverage polygon's security. How this process works?

**Comment by zivkovicmilos** [comment]
cc @Kourin1996

**Reply by Kvenessa** [reply]
@zivkovicmilos, @Kourin1996   any update on this?

**Comment by Kvenessa** [comment]
This announcement article https://blog.polygon.technology/announcing-polygon-sdk-the-gateway-to-multi-chain-ethereum-8ad580ec387/ states that polygon edge version 1 supports building sidechains and version2 will bring in other polygon scaling options for developers to create polygon based chains 

does this mean currently Polygon Edge doesnt allow to create chains with polygon scaling options and allows only to build
1. Ethereum based private network
2. Sidechains with bridging?

**Comment by Kourin1996** [comment, marked as answer]
@Kvenessa Hello, Thank you for your question. Polygon Edge is a private blockchain with compatible EVM and the features to integrate with the mainchain are under development. Please check the future releases for updates.
